easybuild/easyconfigs/c/chaco/chaco-20160817-foss-2016a-Python-3.5.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'PythonPackage'
+
+name = 'chaco'
+version = '20160817'
+commit_id = 'eeae890'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://code.enthought.com/projects/chaco/'
+description = """Chaco is a Python plotting application toolkit that facilitates
+ writing plotting applications at all levels of complexity,
+ from simple scripts with hard-coded data to large plotting programs
+ with complex data interrelationships and a multitude of interactive tools.
+ While Chaco generates attractive static plots for publication and presentation,
+ it also works well for interactive data visualization and exploration."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = ['https://github.com/enthought/chaco/archive/']
+sources = ['%s.tar.gz' % commit_id]
```

Comment:
@hajgato 4.5.0 is the latest official release, why not use that?

Once we start diverging from official releases, we'll need to stick to it forever, since `20160817` will always be newer than `4.5.0` or any other newer version...